### PR TITLE
US-836_Center-on-substation-in-search-window

### PR DESCRIPTION
**Pull request by JacquesBorsenberger**
Signed-off-by: jacques borsenberger <jacques.borsenberger@rte-france.com>

### CODE REVIEW

File: src/components/network/network-explorer.js [hunk 1]
```diff
@@ -199,9 +198,8 @@ const NetworkExplorer = ({
 
     function onDisplaySubstationFocusHandler(event, substation) {
         event.stopPropagation();
-        if (onSubstationFocus !== null) {
-            onSubstationFocus(substation.id);
-        }
+        console.info('jbo zut', substation);
```
Comment (laurgarn) [inline src/components/network/network-explorer.js:201]: To remove

**Review by laurgarn — CHANGES_REQUESTED** [review]
OK for me except the console sig

File: src/components/network/network-explorer.js (hunk 1 above)
Comment (JacquesBorsenberger) [reply src/components/network/network-explorer.js]: done, but PR still in WIP as it need some change in commons ui and though migration to MUI V5

**Review by laurgarn — APPROVED** [review]

**Review by jonenst — DISMISSED** [review]
fix package-lock.json

File: src/components/app-top-bar.js [hunk 2]
```diff
@@ -187,10 +221,15 @@ const AppTopBar = ({ user, tabIndex, onChangeTab, userManager }) => {
                 onSearchTermChange={searchMatchingEquipments}
                 onSelectionChange={showVoltageLevelDiagram}
                 elementsFound={equipmentsFound}
-                renderElement={renderEquipmentForSearchBar(
-                    equipmentClasses,
-                    intl
-                )}
+                renderElement={(props) => {
+                    return (
```
Comment (sBouzols) [inline src/components/app-top-bar.js:225]: remove this return here for simplification and switch '{' to '('

File: src/redux/reducer.js [hunk 3]
```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
```
Comment (sBouzols) [inline src/redux/reducer.js:2]: hum nope

File: src/redux/actions.js [hunk 4]
```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
```
Comment (sBouzols) [inline src/redux/actions.js:2]: 2022 was ok...

File: src/components/study-pane.js [hunk 5]
```diff
@@ -1,11 +1,11 @@
 /**
- * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
```
Comment (sBouzols) [inline src/components/study-pane.js:2]: why not 2022 ?

File: src/components/app-top-bar.js [hunk 6]
```diff
@@ -33,25 +34,58 @@ import makeStyles from '@mui/styles/makeStyles';
...
 const STUDY_VIEWS = [
     StudyView.MAP,
     StudyView.SPREADSHEET,
     StudyView.RESULTS,
     StudyView.LOGS,
 ];
 
+const useEquipmentStyles = makeStyles(equipmentStyles);
+
+const CustomTagRenderer = ({ props, element }) => {
```
Comment (sBouzols) [inline src/components/app-top-bar.js:58]: -> `CustomSuffixRenderer` :smile_cat:  ?

**Review by sBouzols — COMMENTED** [review]
Some comments, I will test

File: src/components/app-top-bar.js (hunk 6 above)
Comment (JacquesBorsenberger) [reply src/components/app-top-bar.js]: done

File: src/components/study-pane.js (hunk 5 above)
Comment (JacquesBorsenberger) [reply src/components/study-pane.js:2]: done

File: src/redux/actions.js (hunk 4 above)
Comment (JacquesBorsenberger) [reply src/redux/actions.js]: done

File: src/redux/reducer.js (hunk 3 above)
Comment (JacquesBorsenberger) [reply src/redux/reducer.js]: done

File: src/components/app-top-bar.js (hunk 2 above)
Comment (JacquesBorsenberger) [reply src/components/app-top-bar.js]: done

**Review by sBouzols — APPROVED** [review]
tests ok

**Pull request merged**